modules/server/src/main/ts/bedrock/compiler/Webpack.ts

```diff
@@ -150,9 +150,15 @@ const getWebPackConfigTs = (tsConfigFile: string, scratchFile: string, dest: str
           }
         }
       }),
-      // See https://github.com/TypeStrong/ts-loader#usage-with-webpack-watch
       new webpack.WatchIgnorePlugin({
-        paths: [ /\.d\.ts$/ ]
+        paths: [
+          // Ignore generated files. See https://github.com/TypeStrong/ts-loader#usage-with-webpack-watch
+          /\.d\.ts$/,
+          /\.(js|ts)\.map$/,
+          // Something seems to trigger that node module package.json files change when they
+          // haven't, so lets just ignore them entirely
+          /node_modules\/.*package\.json$/
```

Comment:
Super picky but should this be `/node_modules\/.*\/package\.json$/` so that it only matches `package.json` not `foo-package.json` files.

Reply:
Oops... I had fixed that in one place (TinyMCE) but not the other it seems :cry: Thanks for the catch Spocke!

Reply:
```suggestion
          /node_modules\/.*\/package\.json$/
```